src/sentry/analytics/events/sentry_app_deleted.py

```diff
@@ -0,0 +1,15 @@
+from __future__ import absolute_import
+
+from sentry import analytics
+
+
+class SentryAppDeletedEvent(analytics.Event):
+    type = 'sentry_app.deleted'
+
+    attributes = (
```

Comment:
is there a reason we don't want the `organization_id` for this event? (for update app as well)

Reply:
We can derive the Org from the `SentryApp`, so I figured we didn't need it here.